messages/data.export.bulk.md

```diff
@@ -1,37 +1,30 @@
 # summary
 
-Bulk export records from an org.
+Bulk export records from an org into a file using a SOQL query. Uses Bulk API 2.0.
 
 # description
 
-Specify a SOQL query to export fields of an object and get them in CSV or JSON format.
+You can use this command to export millions of records from an org, either to migrate data or to back it up.
 
-Use this command to export thousands of records from an org, either to migrate data or backup.
+Use a SOQL query to specify the fields of a standard or custom object that you want to export. Specify the SOQL query either at the command line with the --query flag or read it from a file with the --query-file flag; you can't specify both flags. The --output-file flag is required, which means you can only write the records to a file, in either CSV or JSON format. 
```

Comment:
@cristiand391  -- better?  My aim was to make sure folks understood they can only write to a file -- they can't display the results in the terminal like other commands.   But if you think that's obvious, I'm also okay if you just remove that last sentence completely.

Reply:
Oh I see, when we planned these commands we thought about them like:
1. use `sf data query` when you want to see records printed/play with SOQL
2. use `sf data export bulk` when you already got the right SOQL query

We could name `sf data query` as an alternative here 👍🏼

Reply:
Nah, I think it's fine not to mention `sf data query` -- that command is for a different use case.